src/CMakeLists.txt

```diff
@@ -102,7 +102,14 @@ install(
   LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
-### Package config
+# Copy the tables directory to the install area if the library builds are successful
+add_custom_command(
+  TARGET ${LIB_TARGETS}
+  POST_BUILD
+  COMMENT "Copying tables directory to install area"
+  COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_SOURCE_DIR}/tables ${CMAKE_INSTALL_PREFIX}/tables)
```

Comment:
I should have caught this at review of this PR.
This needs to be fixed with an `install` command.
For testing, we need to link the tables from the source directory to the build directory.